packages/@okta/i18n/src/properties/login.properties

```diff
@@ -932,10 +932,11 @@ oie.phone.invalid = Invalid Phone Number.
 oie.email.label = Email
 oie.email.authenticator.description = Verify with a link or code sent to your email
 oie.email.mfa.title = Verify with your email
+oie.email.challenge.mfa.title = Get a verification email
 oie.email.verify.primaryButton = Send me an email
 # {0} is a user's email address
-oie.email.verify.subtitleWithEmailAddress = Verify with an email link or enter a code sent to <$1>{0}</$1>.
-oie.email.verify.subtitleWithoutEmailAddress = Verify with an email link or enter a code sent to your email.
+oie.email.verify.subtitleWithEmailAddress = Send a verification email to <$1>{0}</$1> by clicking on "Send me an email".
+oie.email.verify.subtitleWithoutEmailAddress = Send a verification email by clicking on "Send me an email".
```

Comment:
Think we should be creating new keys here for the updated strings, [based on this](https://oktawiki.atlassian.net/wiki/spaces/eng/pages/741293778/Requesting+and+Receiving+Translations+i18n+and+l10n#Editing-strings)

Reply:
Done